metpy/calc/tools.py

```diff
@@ -1155,13 +1170,14 @@ def parse_angle(input_dir):
         The angle in degrees
 
     """
-    if len(input_dir) > 1 and not isinstance(input_dir, str):
-        return [parse_angle(single_dir) for single_dir in input_dir]
-    else:
-        input_dir = _abbrieviate_direction(input_dir)
-        for i, dir_str in enumerate(DIR_STRS):
-            if input_dir.upper() == dir_str:
-                return i * BASE_DEGREE_MULTIPLIER
+    if isinstance(input_dir, str):
+        # abb_dirs = abbrieviated directions
```

Comment:
Commented line left in.

Reply:
I think that's actually a descriptive comment.